app/models/application_work_history_break.rb

```diff
@@ -1,16 +1,18 @@
 class ApplicationWorkHistoryBreak < ApplicationRecord
-  include TouchApplicationChoices
-
-  belongs_to :application_form, touch: true, optional: true
-  belongs_to :breakable, polymorphic: true
+  belongs_to :breakable, polymorphic: true, touch: true
 
   before_save -> { self.application_form_id = breakable_id }, if: -> { application_form_id.nil? }
 
-  audited associated_with: :application_form
+  audited associated_with: :breakable
+
+  after_commit do
+    if application_form
```

Comment:
How about `if application_form_breakable`?

Reply:
I think `application_form` is a better method name as it's a public method that can be used by other things, `application_work_history_break.application_form` you just want the application form, you don't really care how it's linked to it

Reply:
Sounds good 👍🏼